components/switch/README.md

```diff
@@ -110,3 +102,7 @@ If an activity is set to `Visible` but also has other conditions affecting its v
   </ul>
 </d2l-switch-visibility>
 ```
+
+## Accessbility
+
+- When `text-position` is set to `hidden`, the switch will still make use of the `text` property as it is used to set the [`aria-label`](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6) of the component, so that screen readers can still pick up what the text of the component is.
```

Comment:
Perhaps we should avoid referring to the internal implementation (i.e. Aria usage), since it's always subject to change.

```suggestion
- When `text-position` is set to `hidden`, the required `text` will be used for the switch's hidden label so that it is accessible to non-sighted users.
```

Reply:
So I was thinking of either adding a short sentence after this one that will include a link to https://www.w3.org/WAI/WCAG22/quickref/#name-role-value, mentioning that this is what it's helping us meet or alternatively add that link to the word `accessible` in this sentence instead. Do you think it's too extra and should just leave things as-is?